static/js/error_popups.js

```
@@ -0,0 +1,13 @@
+function showNotification(id) {
+    const allIds = ['hydeParkWarning', 'noMatchError', 'serverError'];
+    allIds.forEach(hideNotification);
+  
+    const el = document.getElementById(id);
+    if (el) el.classList.remove('is-hidden');
+  }
+  
+  function hideNotification(id) {
```

Instead of hiding the error message, this could create a pop-up in the center of the screen and disable the rest of the screen using an overlay. I think this is a fairly common pattern that's usually implemented by reactive JavaScript libraries.

I found an [example code block](https://www.geeksforgeeks.org/how-to-design-a-custom-alert-box-using-javascript/) that does this. You could add some custom CSS to style the box and avoid using the system prompt here.